translations/default/AdminCatalogFeature.xlf

```diff
@@ -1311,153 +1280,158 @@
   </file>
   <file original="src/PrestaShopBundle/Form/Admin/Product/ProductPrice.php" source-language="en-US" target-language="en" datatype="plaintext">
     <body>
-      <trans-unit id="6f3455d187a23443796efdcbe044096b">
-        <source>No tax</source>
-        <target>No tax</target>
-        <note>Line: 155</note>
-      </trans-unit>
       <trans-unit id="c265e7bd60b9bf0470d39a8ef87e0727">
         <source>Ecotax (tax incl.)</source>
         <target>Ecotax (tax incl.)</target>
-        <note>Line: 186</note>
+        <note>Line: 172</note>
       </trans-unit>
       <trans-unit id="133bc9aa44eaaf809041c01ea1361a07">
         <source>Display the "On sale!" flag on the product page, and on product listings.</source>
         <target>Display the "On sale!" flag on the product page, and on product listings.</target>
-        <note>Line: 220</note>
-      </trans-unit>
-      <trans-unit id="1ea8a394583d38bd981b70c7dcebb104">
-        <source>Price per unit (tax excl.)</source>
-        <target>Price per unit (tax excl.)</target>
-        <note>Line: 174</note>
-      </trans-unit>
-      <trans-unit id="48881913e6addc0c3c439f02229ac91e">
-        <source>Apply to all products</source>
-        <target>Apply to all products</target>
-        <note>Line: 264</note>
-      </trans-unit>
-      <trans-unit id="d9fba514d13913d2b0c9cccfe0e2ecfb">
-        <source>Retail price (tax excl.)</source>
-        <target>Retail price (tax excl.)</target>
-        <note>Line: 162</note>
+        <note>Line: 206</note>
       </trans-unit>
       <trans-unit id="8ea51c95fe66a021471e31fda9053e02">
         <source>Retail price (tax incl.)</source>
         <target>Retail price (tax incl.)</target>
-        <note>Line: 177</note>
+        <note>Line: 163</note>
       </trans-unit>
       <trans-unit id="2028176b385d7a0be3318aedca0bc08d">
         <source>Retail price per unit (tax excl.)</source>
         <target>Retail price per unit (tax excl.)</target>
-        <note>Line: 240</note>
+        <note>Line: 226</note>
+      </trans-unit>
+      <trans-unit id="6f3455d187a23443796efdcbe044096b">
+        <source>No tax</source>
+        <target>No tax</target>
+        <note>Line: 141</note>
+      </trans-unit>
+      <trans-unit id="48881913e6addc0c3c439f02229ac91e">
+        <source>Apply to all products</source>
+        <target>Apply to all products</target>
+        <note>Line: 250</note>
       </trans-unit>
     </body>
   </file>
   <file original="src/PrestaShopBundle/Form/Admin/Product/ProductQuantity.php" source-language="en-US" target-language="en" datatype="plaintext">
     <body>
+      <trans-unit id="a532f6edbb8880c752b74073710b285a">
+        <source>Label when out of stock (and back order allowed)</source>
+        <target>Label when out of stock (and back order allowed)</target>
+        <note>Line: 238</note>
+      </trans-unit>
+      <trans-unit id="5ce00f66be3fd54e40eab844c446e80d">
+        <source>When out of stock</source>
```

Comment:
Where does it appear? It looks the same as the one above.

Reply:
In the product page V2, when creating a new product, in the "Stock" tab